configs/memtier/values.yaml

```diff
@@ -0,0 +1,3 @@
+threads: 1
+type: memcached
+iterations: 1
```

Comment:
```suggestion
iterations: 1

# nodeSelector:
#   role: benchmark

# tolerations:
# - key: role
#   operator: Equal
#   value: benchmark
#   effect: NoSchedule

```

Reply:
Above is creating tables for benchmarks. It is provided by `sysbench prepare`. We need to create tables and delete for benchmarks to run everytime.